cli/lsp/diagnostics.rs

```diff
@@ -1429,18 +1430,7 @@ impl DenoDiagnostic {
         (lsp::DiagnosticSeverity::ERROR, no_local_message(specifier, sloppy_resolution.as_ref().map(|(resolved, sloppy_reason)| sloppy_reason.suggestion_message_for_specifier(resolved))), data)
       },
       Self::ResolutionError(err) => {
-        let mut message;
-        message = enhanced_resolution_error_message(err);
-        if let deno_graph::ResolutionError::ResolverError {error, ..} = err{
-          if let ResolveError::ImportMap(importmap) = (*error).as_ref() {
-            if let ImportMapErrorKind::UnmappedBareSpecifier(specifier, _) = &**importmap {
-              if specifier.chars().next().unwrap_or('\0') == '@'{
-                let hint = format!("\nHint: Use [deno add {}] to add the dependency.", specifier);
-                message.push_str(hint.as_str());
-              }
-            }
-          }
-        }
+        let message = strip_ansi_codes(&enhanced_resolution_error_message(err)).into_owned();
```

Comment:
Before it had the ansi codes sometimes
<img width="348" height="105" alt="image" src="https://github.com/user-attachments/assets/06bd778a-a65b-4f9f-981e-e7909da4c06b" />